test/inproc.js

```diff
@@ -15,6 +15,8 @@ test('inproc socket pub sub', function (t) {
     var addr = 'inproc://pubsub';
     var msg = 'hello world';
 
+    sub.chan('');
```

Comment:
Ah, I meant can we remove all references to the `''` chan from tests that don't test filtering on a sub socket?  Will the sub socket fail to work, since I think we always have to at least subscribe to the empty chan?  If so, since we can have multiple subscription, then we should call `this.chan('')` during socket construction if it's a sub socket.  Otherwise, it's silly to force users to have to explicitly subscribe to the empty channel just to use sub sockets.